### PR TITLE
add search input

### PR DESCRIPTION
**Pull request by safwanAzman**
add search input

### CODE REVIEW

File: resources/views/livewire/sys-admin/permission.blade.php
Comment (nazirulfitrifauzi) [inline resources/views/livewire/sys-admin/permission.blade.php:1]: @safwanAzman Kenape ko comment kn pagination die?

**Review by nazirulfitrifauzi — CHANGES_REQUESTED** [review]
buang yang ko komen kat permision.php & buang komen pagination links tu skli..

File: resources/views/livewire/sys-admin/permission.blade.php
Comment (safwanAzman) [reply resources/views/livewire/sys-admin/permission.blade.php:1]: sebab aku nk tgk full hahaha

**Review by nazirulfitrifauzi — APPROVED** [review]
Done for now

File: resources/views/livewire/sys-admin/permission.blade.php
Comment (safwanAzman) [reply resources/views/livewire/sys-admin/permission.blade.php:1]: sorry do

**Pull request merged**